eve-api-data/src/main/java/com/eveonline/api/config/RepositoryConfiguration.java

```diff
@@ -1,15 +1,15 @@
 package com.eveonline.api.config;
 
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.orm.jpa.EntityScan;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 import org.springframework.transaction.annotation.EnableTransactionManagement;
 
+import com.eveonline.api.repo.ApiDataRepositoryPackage;
+
 @Configuration
-@EnableAutoConfiguration
 @EntityScan(basePackages = {"com.eveonline.api"})
-@EnableJpaRepositories(basePackages = {"com.eveonline.api.repo"})
+@EnableJpaRepositories(basePackageClasses = ApiDataRepositoryPackage.class)
```

Comment:
This is preferable (at least to me) over a string so that refactorings can happen with relative ease.